submissions/editor_guide.md

```diff
@@ -54,7 +48,7 @@ When these sources of information are not enough:
 * look for users of the package or of the data source/upstream service the package connects to (via their opening issues in the repository, starring it, citing it in papers, talking about it on Twitter).
 * You can also search for authors/maintainers of related packages on [PyPI](https://pypi.org/search/).
```

Comment:
we can ask on pyopensci twitter (i've had great success with this approach so far!! )